src/test/java/org/shikshalokam/backend/scp/TestScpPermissionCRUDOperations.java

```diff
@@ -257,4 +290,36 @@ private Response getPermissions(Boolean validToken) {
         X_AUTH_TOKEN=temp_X_AUTH_TOKEN;
         return local;
     }
-}
\ No newline at end of file
+    // Method to delete permission by ID
+    private Response deletePermissionById(int createdId) {
+        try {
+             deletePermissionEndpoint = new URI(PROP_LIST.get("scp.delete.permission.endpoint").toString() + createdId);
```

Comment:
use built in method  get URI.
createrId should be passed in response avoid string concatenation